discovery-provider/src/tasks/entity_manager/user.py

```diff
@@ -168,6 +168,8 @@ def create_user(params: ManageEntityParameters, cid_type: Dict[str, str], cid_me
         )
         cid_type[metadata_cid] = metadata_type
         cid_metadata[metadata_cid] = params.metadata
+        user_record.metadata_multihash = metadata_cid
+        user_record = update_legacy_user_images(user_record)
```

Comment:
nit: this flow will only happen for storage v2 users right? maybe it's better to remove this line and stop using `update_legacy_user_images` going forward, since all the v2 user stuff only sets the correct `_sizes` fields and not the legacy ones